packages/components/nav/src/nav.tsx

```diff
@@ -2,7 +2,15 @@ import styled from "styled-components";
 
 import { Flex, FlexProps } from "@buoysoftware/anchor-layout";
 
-export const Nav = styled((props) => {
+type Variant = "main" | "sub";
+
+interface OwnProps {
+  variant?: Variant;
```

Comment:
Is this the first introduction of a variant? If so, should we consider abandoning the term and make a separate component for SubNav?

Reply:
It is but I'm working on buttons which will likely have variants as well.

Reply:
It also feels like a good use for a variant here, where everything is the same except some minor styling differences. As opposed to our old text component which got to complicated with variants etc.

Reply:
Seems reasonable to me just wanted to make sure.